ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/RpcMethod.java

```diff
@@ -49,7 +49,6 @@ public enum RpcMethod {
   ENGINE_NEW_PAYLOAD("engine_newPayloadV1"),
   ENGINE_FORKCHOICE_UPDATED("engine_forkchoiceUpdatedV1"),
   ENGINE_EXCHANGE_TRANSITION_CONFIGURATION("engine_exchangeTransitionConfigurationV1"),
-  ENGINE_PREPARE_PAYLOAD("engine_preparePayload_debug"),
```

Comment:
?

Reply:
the RC3 branch originally had #4427 merged in to help testing async block production.

Reply:
👍